webroot/src/models/State/State.model.ts

```diff
@@ -38,7 +38,17 @@ export class State implements InterfaceStateCreate {
     // Helper methods
     public name(): string {
         const abbrev = (this.abbrev || '').toUpperCase() || '';
-        const states = this.$tm('common.states') || [];
+        let states = this.$tm('common.states') || [];
+
+        if (typeof states[0]?.abbrev === 'function') {
+            const normalize = ([value]) => value;
+
+            states = states.map((st) => ({
+                abbrev: st.abbrev({ normalize }),
+                full: st.full({ normalize }),
+            }));
+        }
```

Comment:
What's going on here? (I understand step-by-step what's happening, but wondering why it needs to be done, and why `abbrev` might be something other than a string.)

Same question on `StateUpload.ts` but I assume the answer is the same/similar.

Reply:
Good question. It's related to the previous item about the separate runtime-only version of vue-i18n. Their current **non**-runtime-only version has an issue with CSP; and their build bundling is a little chaotic. It was a decision to stick with that library for now, as they claim to have addressed these things in their latest Beta version. Once they've gone stable with that new version, we can evaluate if we want to stick with it, or migrate away.

Reply:
Got it. Can we make a backlog item that references the relevant code, including this code and that in `StateUpload.ts`, to make an update when the library is updated or if we decide to change translation libraries?

Reply:
Sounds good. I created #118 to track this.